### PR TITLE
[DAT-511] - Add endpoint to get billing information 

### PR DESCRIPTION
**Pull request by bamaro-dev**
Here is added a new endpoint, in order to get the user billing information:
```
{
  firstname: string;
  lastname: string;
  address: string;
  city: string;
  province: string;
  country: string;
  zipCode: string;
  phone: string;
}
```

*We used a JSON for the security questions that it was save in cdn:
- ES: https://cdn.fromdoppler.com/static-data/questions-es.json
- EN: https://cdn.fromdoppler.com/static-data/questions-en.json

### CODE REVIEW

File: Doppler.BillingUser/Controllers/BillingController.cs [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
+        private readonly BillingRepository _billingRepository;
+
+        public BillingController(ILogger<BillingController> logger, BillingRepository billingRepository)
         {
-            return $"Hello! \"you\" that have access to GetCurrentBillingInformation with accountname '{accountname}'";
+            _logger = logger;
+            _billingRepository = billingRepository;
+        }
+
+        //[Authorize(Policies.OWN_RESOURCE_OR_SUPERUSER)]
```
Comment (nreal) [inline Doppler.BillingUser/Controllers/BillingController.cs:23]: I think that this line should not be comment

Comment (bamaro-dev) [reply Doppler.BillingUser/Controllers/BillingController.cs:23]: You right!

File: Doppler.BillingUser/Infrastructure/BillingRepository.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        public async Task<BillingInformation> GetBillingInformation(string email)
+        {
+            using (IDbConnection connection = await _connectionFactory.GetConnection())
+            {
+                var results = await connection.QueryAsync<BillingInformation>(@"
+SELECT
+    U.BillingFirstName AS Firstname,
+    U.BillingLastName AS Lastname,
+    U.Address,
+    U.CityName AS City,
```
Comment (nreal) [inline Doppler.BillingUser/Infrastructure/BillingRepository.cs:25]: In the User table exist a column with the name [BillingCity], maybe this column should be replaced for it. What do you think?

File: Doppler.BillingUser/Infrastructure/BillingRepository.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        }
+        public async Task<BillingInformation> GetBillingInformation(string email)
+        {
+            using (IDbConnection connection = await _connectionFactory.GetConnection())
+            {
+                var results = await connection.QueryAsync<BillingInformation>(@"
+SELECT
+    U.BillingFirstName AS Firstname,
+    U.BillingLastName AS Lastname,
+    U.Address,
```
Comment (nreal) [inline Doppler.BillingUser/Infrastructure/BillingRepository.cs:24]: In the User table exist a column with the name [BillingAddress], maybe this column should be replaced for it. What do you think?

File: Doppler.BillingUser/Infrastructure/BillingRepository.cs (hunk 2 above)
Comment (bamaro-dev) [reply Doppler.BillingUser/Infrastructure/BillingRepository.cs:25]: Yes, I realize about this after push changes, I already corrected it. Thanks!

File: Doppler.BillingUser/Infrastructure/BillingRepository.cs (hunk 3 above)
Comment (bamaro-dev) [reply Doppler.BillingUser/Infrastructure/BillingRepository.cs:24]: Yes, I realize about this after push changes, I already corrected it. Thanks!

**Comment by nreal** [comment]
👍

File: Doppler.BillingUser/Infrastructure/BillingRepository.cs [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        {
+            using (IDbConnection connection = await _connectionFactory.GetConnection())
+            {
+                var results = await connection.QueryAsync<BillingInformation>(@"
+SELECT
+    U.BillingFirstName AS Firstname,
+    U.BillingLastName AS Lastname,
+    U.BillingAddress AS Address,
+    U.BillingCity AS City,
+    isnull(S.Name, '') AS Province,
```
Comment (nreal) [inline Doppler.BillingUser/Infrastructure/BillingRepository.cs:26]: It is ok that we take the **province** from User?

Comment (bamaro-dev) [reply Doppler.BillingUser/Infrastructure/BillingRepository.cs:26]: I'm using `IdBillingState `into the `WHERE`. Your doubt is about that?

Comment (nreal) [reply Doppler.BillingUser/Infrastructure/BillingRepository.cs:26]: It is ok, thank  I found it

**Pull request merged**